### PR TITLE
Issues/330 - Adds Run 2.1i dr1b catalog.

### LINKED ISSUE #330: Create Run 2.1i Object Table catalog config/reader for 'drb1 '
DR1b has been kindly prepared by @johannct  and transferred to NERSC by @airnandez 
https://github.com/LSSTDESC/ImageProcessingPipelines/issues/86#issuecomment-504722527

We can and should quickly make a catalog config for it and make it available through GCR.
/global/projecta/projectdirs/lsst/global/in2p3/Run2.1i/w_2019_19-v1/dpdd/calexp-v1:coadd-dr1b-v1/object_table_summary

### PR DESCRIPTION
**Pull request by wmwv**
Adds
I created and ran the `trim_tract_cat.py` to generate the trimmed files.
See new Notebook `validate_dc2_run2.1i_object_table.ipynb` in `DC2-analysis` branch `issues/95` for a quick tour of the Object Table.

https://github.com/LSSTDESC/DC2-analysis/blob/issues/95/validation/validate_dc2_run2.1i_object_table.ipynb

[edited by @yymao: fix #330]

### CODE REVIEW

File: GCRCatalogs/catalog_configs/dc2_object_run2.1i_dr1a_tract4850.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+subclass_name: dc2_object.DC2ObjectCatalog
+base_dir: /global/projecta/projectdirs/lsst/production/DC2_ImSim/Run2.1i/dpdd/calexp-v1:coadd-v1/object_table_summary
+schema_filename: trim_schema.yaml
+filename_pattern: 'trim_object_tract_4850\.hdf5$'
+description: DC2 Run 2.1i Object Catalog Tract 4850 (with only DPDD columns and native columns needed for the DPDD columns)
+creators: ['DESC DC2 Team']
+pixel_scale: 0.2
+include_in_default_catalog_list: true
```
Comment (yymao) [inline GCRCatalogs/catalog_configs/dc2_object_run2.1i_dr1a_tract4850.yaml]: Do we want to include this catalog in the default list?

File: GCRCatalogs/catalog_configs/dc2_object_run2.1i_dr1b_all_columns.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+subclass_name: dc2_object.DC2ObjectCatalog
+base_dir: /global/projecta/projectdirs/lsst/global/in2p3/Run2.1i/w_2019_19-v1/dpdd/calexp-v1:coadd-dr1b-v1/object_table_summary
+schema_filename: trim_schema.yaml
+filename_pattern: 'object_tract_\d+\.hdf5$'
+description: DC2 Run 2.1i Object Catalog (with only DPDD columns and native columns needed for the DPDD columns)
+creators: ['DESC DC2 Team']
+pixel_scale: 0.2
+include_in_default_catalog_list: true
```
Comment (yymao) [inline GCRCatalogs/catalog_configs/dc2_object_run2.1i_dr1b_all_columns.yaml]: Do we want to include this catalog in the default list?

File: GCRCatalogs/catalog_configs/dc2_object_run2.1i_dr1b_tract4850.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+subclass_name: dc2_object.DC2ObjectCatalog
+base_dir: /global/projecta/projectdirs/lsst/global/in2p3/Run2.1i/w_2019_19-v1/dpdd/calexp-v1:coadd-dr1b-v1/object_table_summary
+schema_filename: trim_schema.yaml
+filename_pattern: 'trim_object_tract_4850\.hdf5$'
+description: DC2 Run 2.1i Object Catalog Tract 4850 (with only DPDD columns and native columns needed for the DPDD columns)
+creators: ['DESC DC2 Team']
+pixel_scale: 0.2
+include_in_default_catalog_list: true
```
Comment (yymao) [inline GCRCatalogs/catalog_configs/dc2_object_run2.1i_dr1b_tract4850.yaml]: Do we want to include this catalog in the default list?

File: GCRCatalogs/catalog_configs/dc2_object_run2.1i_dr1b_all_columns.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+subclass_name: dc2_object.DC2ObjectCatalog
+base_dir: /global/projecta/projectdirs/lsst/global/in2p3/Run2.1i/w_2019_19-v1/dpdd/calexp-v1:coadd-dr1b-v1/object_table_summary
+schema_filename: trim_schema.yaml
```
Comment (yymao) [inline GCRCatalogs/catalog_configs/dc2_object_run2.1i_dr1b_all_columns.yaml]: Shouldn't this be `schema.yaml`?

**Comment by yymao** [comment]
@wmwv in addition, we probably should 
- Update README.md (dr1a -> dr1b)
- Remove `include_in_default_catalog_list` in dr1a configs?

File: GCRCatalogs/catalog_configs/dc2_object_run2.1i_dr1a_tract4850.yaml (hunk 1 above)
Comment (wmwv) [reply GCRCatalogs/catalog_configs/dc2_object_run2.1i_dr1a_tract4850.yaml]: No.  Fixed.

File: GCRCatalogs/catalog_configs/dc2_object_run2.1i_dr1b_all_columns.yaml (hunk 2 above)
Comment (wmwv) [reply GCRCatalogs/catalog_configs/dc2_object_run2.1i_dr1b_all_columns.yaml]: No.  Fixed.

**Comment by yymao** [comment]
@wmwv the paths need to be updated, according to https://github.com/LSSTDESC/gcr-catalogs/issues/330#issuecomment-507764452, right?

**Comment by wmwv** [comment]
Agreed, was just waiting on the completion of the copy, which just finished.

**Comment by wmwv** [comment]
Paths updated and inspection notebook rerun to verify new paths.

**Pull request merged**